test/apps/iodemo/io_demo.cc

```diff
@@ -494,12 +513,20 @@ class DemoClient : public P2pDemoCommon {
 
         if (hdr->op == IO_COMP) {
             ++_num_completed;
+            int index = get_conn_index(conn);
+            ++_conn[index].num_completed;
```

Comment:
@evgeny-leksikov ``++_conn[get_conn_index(conn)].num_completed;`` Fixed.
Can't use ``++conn->num_completed;`` since conn is not the same type as _conn[get_conn_index(conn)].